Makefile

```diff
@@ -49,9 +86,15 @@ clear:
 lib-native: Cargo.lock wgpu-native/Cargo.toml $(WILDCARD_WGPU_NATIVE)
 	cargo build --manifest-path wgpu-native/Cargo.toml
 
+lib-native-release: Cargo.lock wgpu-native/Cargo.toml $(WILDCARD_WGPU_NATIVE)
+	cargo build --manifest-path wgpu-native/Cargo.toml --release
+
 lib-remote: Cargo.lock wgpu-remote/Cargo.toml $(WILDCARD_WGPU_REMOTE)
 	cargo build --manifest-path wgpu-remote/Cargo.toml
 
+lib-remote-release: Cargo.lock wgpu-remote/Cargo.toml $(WILDCARD_WGPU_REMOTE)
```

Comment:
Let's not bother with packaging the "lib-remote" thing, at least for now. It's not clear how this will be evolving for now.

Reply:
Alright, I'll take it out. I wasn't really sure about that.